### PR TITLE
Model Config Documents Quality Improvement[mnasnet, mobilenetv1, mobilenetv2, mobilenetv3]

### PR DESCRIPTION
**Pull request by The-truthh**

### CODE REVIEW

File: configs/mnasnet/README.md [hunk 1]
```diff
@@ -2,11 +2,84 @@
...
+python validate.py -c configs/mnasnet/mnasnet_0.75_ascend.yaml --data_dir /path/to/imagenet --ckpt_path /path/to/ckpt
+```
+
+### Deployment
+
+Please refer to the [deployment tutorial](https://github.com/mindspore-lab/mindcv/blob/main/tutorials/deployment.md) in MindCV.
+
+## References
+
+<!--- Guideline: Citation format should follow GB/T 7714. -->
```
Comment (geniuspatrick) [inline configs/mnasnet/README.md:84]: 这行是注释，可以删掉

File: configs/mobilenetv1/README.md [hunk 2]
```diff
@@ -1,20 +1,86 @@
...
+python validate.py -c configs/mobilenetv1/mobilenet_v1_0.25_ascend.yaml --data_dir /path/to/imagenet --ckpt_path /path/to/ckpt
+```
+
+### Deployment
+
+Please refer to the [deployment tutorial](https://github.com/mindspore-lab/mindcv/blob/main/tutorials/deployment.md) in MindCV.
+
+## References
+
+<!--- Guideline: Citation format should follow GB/T 7714. -->
```
Comment (geniuspatrick) [inline configs/mobilenetv1/README.md:85]: 同上

**Comment by SamitHuang** [comment]
mnastnet and mobilenet v1 are viewed and passed 👍

**Comment by geniuspatrick** [comment]
Temporarily closed。

**Comment by Songyuanwei** [comment]
#334

**Pull request merged**